### ISSUE TITLE
About how to install

### ISSUE THREAD

**Issue by xiejing1006** [hellochar/Shadow-of-the-Honey-Badger#1]
Hi, I see your shadow honeypots project in github. But the readme is empty, so I am asking is the project finished? If it's finished ,can you provide a readme? Thank you~

**Comment by scarletknight11** [comment]
yes this project is mostly finished. it is actually something my team and i made at global game jam. i can provide you a readme

**Comment by BNarinesingh** [comment]
I went back to the code and was fiddling with it but I didn't finish it, the commit was just for a save point in case I broke something along the way. Good to hear from you guys, I hope all is well.
